.ci/run.sh

```diff
@@ -189,7 +189,11 @@ function run_style() {
 }
 
 function run_commit(){
-  dffml service dev lint commits
+  BRANCH="$(echo $GITHUB_REF | cut -d'/' -f 3)"
+  echo "On Branch: ${BRANCH}"
+  if [[ "$BRANCH" != "master" ]]; then
```

Comment:
```suggestion
  if [[ "x${BRANCH}" != "xmaster" ]]; then
```

Reply:
I didn't do that because I am pretty sure the GitHub Actions probably uses bash and as it is  a POSIX-compliant shell this (afaik) makes the added x comparison redundant 🤔

Reply:
However if there's any other reason to add this then please let me know as well.